db/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM camptocamp/postgres:9.6
-
+# RUN rm -r /var/lib/postgresql/data/*
 ENV POSTGRES_USER test
 ENV POSTGRES_DB test
-ADD *.sql /docker-entrypoint-initdb.d/
+ADD add_dataset_db.sql /docker-entrypoint-initdb.d/add_dataset_db.sql
```

Comment:
maybe move the initial script into a subdirectory instead such as db/init ?

Reply:
Not sure I understand...